cms/static/cms/js/build.js

```diff
@@ -171,5 +171,4 @@
          */
         logLevel: 1
     };
-}())  // eslint-disable-line semi
-// A semicolon on the line above will break the requirejs optimizer
+}());
```

Comment:
I added that comment when I ran the eslint autofixer on the codebase. It was almost unbelievable to me that a semicolon there would completely break the build - because there definitely _should_ be a semicolon there - so if under this new version tests pass with a semicolon, go for it.